R/Model-class.R

```diff
@@ -3335,3 +3335,98 @@ FractionalCRM <- function(...) {
     sigma2 = 2
   )
 }
+
+## class ----
+
+#' `LogisticLogNormalOrdinal`
+#'
+#' @description `r lifecycle::badge("experimental")`
+#'
+#' [`LogisticLogNormalOrdinal`] is the class for a logistic lognormal CRM model
+#' using an ordinal toxicity scale.
+#'
+#' @aliases LogisticLogNormalOrdinal
+#' @export
+.LogisticLogNormalOrdinal <- setClass(
+  Class = "LogisticLogNormalOrdinal",
+  contains = "ModelLogNormal",
+  slots = c(datablock = "function")
+  # ,
+  # validity = v_model_ordinal_logistic_log_normal
```

Comment:
Would be good to have the validation function?

Reply:
The `datablock` slot is a left over from a previous iteration of the design.  It's not necessary and I have deleted it.  Therefore, I think there is no need for a specific validation function for `LogisticLogNormalOrdinal`.  Do you agree?

Reply:
yep sounds good - then just delete the commented code